### ISSUE TITLE
OverqualifiedSelector linter

### ISSUE THREAD

**Issue by davidtheclark** [brigade/scss-lint#267]
Proposal:

This linter would disallow just one thing: Qualifying class, attribute, or, of course, id selectors with a tag.

Bad:

``` scss
div#thing { ... }
ul.list { ... }
ul li.item { ... }
a[href="place"] { ... }
```

Good:

``` scss
#thing { ... }
.list { ... }
ul .item { ... }
[href="place"] { ... }
```

Ideally you could avoid situations altogether where you really felt compelled to qualify with a tag. But maybe situations arise and you feel you have to ... it's not like the practice is 100% anathema ... so that's where https://github.com/causes/scss-lint/issues/150 would have to come in ...
